docs/blog/2021-09-28-building-a-startup-with-rust.md

```diff
@@ -0,0 +1,163 @@
+the full picture.
+
+In our experience the competence of your average Rust developer is much higher
+than more conventional programming languages. Something spoke to these
+individuals when they picked up Rust, and it's hard to put your finger on it but
+it's that same quality that makes a great engineer. It's also been a pleasant
+surprise to find out that really good engineers will seek you out as an
+employer *because you use Rust*. They don't want to work in *script or Java or
+C++. They want to work with Rust because it's great.
+
+## Open Source
+
+At Synth, we've chosen to adopt an open-core business model. The idea behind an
+open-core business is you develop and open source product with a permissive
+license which solves a real *technical* problem. You work on building a user
+base, a community and a great product all out in the open. You then structure
+your business model around solving the corresponding *organisational* problem -
+and that's how you make money.
+
+We've been really lucky to have a really active set of contributors - giving
+ideas, reporting bugs and contributing (at times very significant) code. It is
+hard to know for sure, but we have a strong hunch that a lot of the contributors
+are active because they have an interest in Rust projects specifically. A lot of
+our contributors are also interested in learning Rust - not necessarily being
+veterans of the language. This has worked out great as the more experienced
+members of our core team mentor and review code of young rustaceans, building a
+symbiotic positive feedback loop.
+
+Thank you to all our contributors - you know who you are and you guys are
+amazing.
+
+## Libraries
+
+Rust has an ecosystem of incredibly high quality libraries. The Rust core team
+has led by example and focused on an incredibly high quality and tight standard
+library. The result of a highly focused standard library is (unfortunately) a
+lack of canonical libraries for doing things outside the standard library. So
+you want a webserver, pick from one of the 100s available. You want a crate (
+Rust lingo for library) for working with JWT tokens? Here's 9, pick one. I mean,
+even something as fundamental as an asynchronous runtime is split
+between `tokio`
+and `async-std`. As a young Rustaecean this can be overwhelming.
+
+What ends up happening over time is certain libraries become implicitly
+canonical as they receive overwhelming support and start becoming serious
+dependecies differentiating from their alternatives. I predict this will get
+better as the ecosystem matures, but predictions are dangerous. // TODO: I think
+niko matsakis talked about them standardising this
+
+The lack of canonical non-standard libraries is an issue when you're getting
+started - but over time this diminishes as you get a better understanding of the
+ecosystem. What *has* been constantly detrimental to our development velocity
+has been the lack of *integration* libraries for Rust. We've had to write a
```

Comment:
We should explain the term "integration library", as it may be unclear to some of our readers.